assets/sass/patterns/molecules/main-menu.scss

```diff
@@ -28,7 +28,7 @@
     }
   }
 
-  &.lineSeparator {
+  &.border-line__bottom {
```

Comment:
Class needs to be renamed to something more relevant. @nlisgo will talk to you about this today

Reply:
The class names have been changed and we have swapped the `hasBorderBottom ` flag with a `classes` property.